open-company/open-company-web
# Follow-up improvements

Card: https://trello.com/c/DEcCHxNB

To test:
- on desktop
- expand quick view
- add title
- add a long body
- click follow-up button
- scroll up and down
- [x] follow-ups blue bar still in place?
- now click fullscreen button
- scroll up and down
- [x] follow-ups bar is sticky?
- now click the blue bar
- [x] the button says Deselect All (not Select All)?
- select yourself also
- [x] the button still says Deselect All?
- click Deselect All
- [x] now it says Select All
- pick some random users
- [x] button still correct (play with selecting/unselecting users and make sure it works)
- make sure there is follow-up for yourself
- publish the post
- expand the post
- focus on add comment
- [x] do you  see the complete follow-up checkbox besides the comment button?
- make sure it's unchecked
- add a comment
- [x] follow-up still uncompleted?
- now click reply to your comment
- [x] do you see the follow-up checkbox there too?
- check it
- send a reply
- [x] follow-up completed?

Mobile test:
- start editing a new post
- add title and a long body
- [x] make sure the blue bar is sticky and doesn't cover any needed part of the screen
